lint/verilator_waiver.vlt

```diff
@@ -10,55 +10,55 @@
 // Otherwise, only global waivers are applied, but not file-specific waivers.
 
 `verilator_config
-lint_off -msg PINCONNECTEMPTY
+lint_off -rule PINCONNECTEMPTY
 
 // Filename 'ibex_register_file_ff' does not match MODULE name: ibex_register_file
 // ibex_register_file_ff and ibex_register_file_latch provide two
 // implementation choices for the same module.
-lint_off -msg DECLFILENAME -file "*/rtl/ibex_register_file_ff.sv"
-lint_off -msg DECLFILENAME -file "*/rtl/ibex_register_file_latch.sv"
+lint_off -rule DECLFILENAME -file "*/rtl/ibex_register_file_ff.sv"
+lint_off -rule DECLFILENAME -file "*/rtl/ibex_register_file_latch.sv"
 
 // Bits of signal are not used: boot_addr_i[7:0]
 // Boot address is 256B aligned, cleaner to pass all bits in
-lint_off -msg UNUSED -file "*/rtl/ibex_if_stage.sv" -lines 40
+lint_off -rule UNUSED -file "*/rtl/ibex_if_stage.sv" -match "'boot_addr_i'[7:0]"
```

Comment:
What are those `'` for? Are there docs somewhere how the matching works? (I thought it's simple wildcard matches.)

Reply:
Oddly, they are part of the output generate by Verilator. It may make sense to look into expanding it to include the dimensions, but its not on my todo list. So, nothing regex specific, but the standard output..

Reply:
I'm pretty sure we copied the comment above the waiver directly from the error message, and it has no `'`. Maybe that's a recent Verilator change? Could either you @wallento or @tomroberts-lowrisc see if that's something that changed between Verilator 4.010 (roughly) and today? I would like to at least ensure that we understand what's going on, and Verilator won't change back to the original formatting in the next release.

Reply:
It was commit 0f857ce423d3498ddf51f7a5955783f07a534767 (between 4.016 and 4.018)
Wilson seems to have done it deliberately so I guess it will stay that way for the foreseeable.

Reply:
Thanks for checking that Tom!